package.json

```diff
@@ -9,9 +9,19 @@
     "react-router-dom": "^4.2.2"
   },
   "devDependencies": {
+    "eslint": "^4.9.0",
```

Comment:
Aren't these already included with CRA?

Reply:
Eslint is included with CRA, but none of the other deps we need for airbnb or prettier plugins.

Reply:
CRA uses eslint:recommended instead (see https://github.com/facebook/create-react-app/issues/3540#issuecomment-358792632), but I think we can avoid this when we bump to react-scripts 2.0 (if we do want airbnb here).
Can we split this eslint/prettier stuff into a separate PR so we can continue there and not hold things up here.